### DISCUSSION TITLE
--memory-swap not supported in lima/nerdctl

### DISCUSSION THREAD

**Discussion by doberkofler** [containerd/nerdctl#1007] [Q&A] [answered]
nerdctl does currently not support the `--memory-swap` option and when using `free -m` in a container it seems as if there would be no swap memory available. Does my container have swap memory in lima/nerdctl?

```bash
              total        used        free      shared  buff/cache   available
Mem:           7945         752        2587        1522        4605        5404
Swap:             0           0           0
```

**Comment by junnplus** [comment, marked as answer]
Since https://github.com/containerd/nerdctl/pull/1110, we supported the `--memory-swap` option in nerdctl.

PS: `free -m` shows information about the host(lima instance) rather than the container. swap 0 means no swap memory available.
